### ISSUE TITLE
financial transparency

### ISSUE THREAD

**Issue by yoshuawuyts** [rust-lang/foundation-faq-2020#40]
Does the foundation intend to share insights into its finances? If so, what can we expect that to look like?

**Comment by nikomatsakis** [comment]
Yes, the Foundation definitely intends to make financial transparency a high priority. You can expect regular reports that discuss what the foundation has done and financial details.

**Comment by yoshuawuyts** [comment]
Ah, I closed this issue to clean up my issue backlog -- I did not realize it was intended to be kept open, I'm guessing for further discussion? Apologies!
